src/main/java/com/sequenceiq/cloudbreak/service/stack/flow/ProvisionContext.java

```diff
@@ -0,0 +1,165 @@
+package com.sequenceiq.cloudbreak.service.stack.flow;
```

Comment:
This class has been removed in the previous PR. (Class has been renamed, logic factored out)